docs_user/modules/proc_migrating-the-rgw-backends.adoc

```diff
@@ -71,24 +69,25 @@ spec:
   rgw_zone: default
 ----
 
-. In the `placement` section, replace the following values:
-* Replace the controller nodes with the `label: rgw` label.
-* Change the ` rgw_frontend_port` value to `8090` to avoid conflicts with the Ceph ingress daemon.
+. In the `placement` section, ensure that the `label` and `rgw_frontend_port` values are set:
 +
 ----
 ---
 networks:
 - 172.17.3.0/24
 placement:
-  label: rgw
+  label: rgw <1>
 service_id: rgw
 service_name: rgw.rgw
 service_type: rgw
 spec:
-  rgw_frontend_port: 8090
+  rgw_frontend_port: 8090 <2>
   rgw_realm: default
   rgw_zone: default
 ----
++
+<1> Replace the Controller nodes with the `label: rgw` label.
+<2> Change the `rgw_frontend_port` value to `8090` to avoid conflicts with the Ceph ingress daemon.
```

Comment:
"Change the `rgw_frontend_port` value to `8090`, to avoid conflicts with the Ceph ingress daemon."

could add a comma here if you wanted